### ISSUE TITLE
Question about loss function

### ISSUE THREAD

**Issue by Jasonlee1995** [Richarizardd/Self-Supervised-ViT-Path#3]
Hi.

Since reading your paper, I'm quite confused about why did you use the loss function like above? or maybe the typos?
(I cannot find your train codes so only way to check is by making issues)

Loss function you used is some weird cause it measures the cross entropy of teacher distribution relative to student distribution and train student network.

Your reference paper DINO use different loss function as below, which measures the cross entropy of student distribution relative to teacher distribution and train student network.

Is there some missing points that I don't know or it's just typos?

**Comment by Richarizardd** [comment]
Hi @Jasonlee1995 - it may have been a typo, but I will have to double check as well. Similar issue on DINO github.

https://github.com/facebookresearch/dino/issues/169
